lua/efmls-configs/formatters/prettier.lua

```diff
@@ -1,9 +1,25 @@
 local fs = require('efmls-configs.fs')
 
 local formatter = 'prettier'
-local command = string.format('%s --stdin --stdin-filepath ${INPUT}', fs.executable(formatter, fs.Scope.NODE))
+local command = string.format(
+  '%s --stdin --stdin-filepath ${INPUT} ${--range-start:charStart} '
+    .. '${--range-end:charEnd} ${--tab-width:tabSize} ${--use-tabs:!insertSpaces}',
```

Comment:
Thanks for the PR!

Are these flags included or ignored when also performing a `textDocument/formatting` request? I did not see any reference to them on your PR (https://github.com/mattn/efm-langserver/pull/246).

Reply:
@creativenull they get removed if the placeholder is filled, so in the case of `textDocument/formatting` it ends up the same as the old command

Reply:
Gotcha 👍🏾